src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -58,7 +58,8 @@ protected function describeRoute(Route $route, array $options = array())
             ."\n".'- Class: '.\get_class($route)
             ."\n".'- Defaults: '.$this->formatRouterConfig($route->getDefaults())
             ."\n".'- Requirements: '.($route->getRequirements() ? $this->formatRouterConfig($route->getRequirements()) : 'NO CUSTOM')
-            ."\n".'- Options: '.$this->formatRouterConfig($route->getOptions());
+            ."\n".'- Options: '.$this->formatRouterConfig($route->getOptions())
+            ."\n".'- Condition: '.('' !== $route->getCondition() ? $route->getCondition() : '');
```

Comment:
I'd suggest not adding this line when "condition" is empty (as done in the json - and same for other descriptors)

Reply:
ok :+1:

Reply:
@nicolas-grekas I added your suggestions :smiley:
it's better now?

Reply:
@nicolas-grekas i'm waiting for your feedback regarding this changes.

Reply:
friendly ping @nicolas-grekas